tests/test_persist_on_failure.py

```diff
@@ -34,10 +34,7 @@ def test_fallible_task_with_failure():
     assert failed_task_object.task_name == tasks.fallible_task.name
     assert failed_task_object.args == []
     assert failed_task_object.kwargs == {'message': 'The example task failed'}
-    if six.PY2:
-        assert failed_task_object.exc == "ValueError(u'The example task failed',)"
-    else:
-        assert failed_task_object.exc == "ValueError('The example task failed',)"
+    assert failed_task_object.exc == "ValueError('The example task failed')"
```

Comment:
I think that we can also do something like this:
```
assert failed_task_object.exc == repr(ValueError('The example task failed'))
```

So we don't have to apply the other changes.

Reply:
Yes this can be done this way